### PR TITLE
Rosetta Deployment Tests

### PR DESCRIPTION
**Pull request by james-iohk**
# Description

Adds automated API checks for Rosetta deployments on testnet, stagning, shelley-qa and mary-qa.

# Proposed Solution

Uses Postman to define API test scenarios and uses Newman to execute them via CLI.

# Important Changes Introduced

N/A

# Testing

Run as described in readme.

### CODE REVIEW

File: .github/workflows/deployment_test.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+name: Deployed Environment Checks
+
+on:
+  schedule:
+    - cron: '0 12 * * *'
+  push:
+    branches:
+    - feat/deployment-tests
```
Comment (james-iohk) [inline .github/workflows/deployment_test.yml:8]: Remove this before merge

File: .github/workflows/deployed_environment-test.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
+name: Deployed Environment Checks
+
+on:
+  schedule:
+    - cron: '0 12 * * *'
+  push:
```
Comment (rhyslbw) [inline .github/workflows/deployed_environment-test.yml:6]: Remove the push condition

File: test/deployed-environment-test/package.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "deployment-test",
+  "version": "1.0.0",
+  "description": "### Description Automated API checks for Rosetta deployments on testnet, stagning, shelley-qa and mary-qa.",
```
Comment (rhyslbw) [inline test/deployed-environment-test/package.json:4]:
```suggestion
  "description": "### Description Automated API checks for Rosetta deployments on testnet, staging, shelley-qa and mary-qa.",
```

**Review by rhyslbw — CHANGES_REQUESTED** [review]
There are newman run reports committed, is that intentional? Also is there a reason we're not checking `mainnet`?

**Comment by james-iohk** [comment]
> There are newman run reports committed, is that intentional? Also is there a reason we're not checking `mainnet`?
1. Woops, nope. Must have slipped in during a final force push.
2. Good shout, I didn't even think we had it deployed there yet.

**Review by rhyslbw — APPROVED** [review]
LGTM. Not sure why the Hydra check failed, as the jobs are showing up as passed.

File: test/deployed-environment-test/README.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+## Deployment Test
+
+### Description
+Automated API checks for Rosetta deployments on testnet, stagning, shelley-qa and mary-qa. Can also be used for local instance.
```
Comment (rhyslbw) [inline test/deployed-environment-test/README.md:4]:
```suggestion
Automated API checks for Rosetta deployments on testnet, staging, shelley-qa and mary-qa. Can also be used for local instance.
```

**Review by rhyslbw — CHANGES_REQUESTED** [review]
Just picked up a minor typo. Fixing this will also re-run the Hydra jobs to see if we need to escalate to the nix team.

**Review by rhyslbw — APPROVED** [review]

**Pull request merged**